modin/core/execution/unidist/implementations/pandas_on_unidist/partitioning/partition.py

```diff
@@ -162,8 +162,8 @@ def wait(self):
         wait([self._data])
 
     # If unidist has not been initialized yet by Modin,
-    # unidist itself handles initialization when calling `unidist.put`.
-    _iloc = unidist.put(PandasDataframePartition._iloc)
+    # it will be initialized when calling `UnidistWrapper.put`.
```

Comment:
```suggestion
    # unidist itself handles initialization when calling `unidist.put`.
```
Looks clearer to me because it might be not obvious for a Modin developer how `UnidistWrapper.put` inititializes unidist (there is no any init call).

Reply:
My version is clearer in terms of what it says about the function that is called on the next line. Your option implies that the developers know that `unidist.put` function is called inside of `UnidistWrapper.put`.

Reply:
`unidist.put` might be called in other places rather than `UnidistWrapper.put` so unidist gets initialized. So I propose to leave as is (revert your changes back) and aslo add `Notes` section to `UnidistWrapper.put` where we will say that when calling `unidist.put` unidist will be initialized if it hasn't been yet.

Reply:
> `unidist.put` might be called in other places rather than `UnidistWrapper.put`

After this pull request, this use is not expected. `UnidistWrapper.put` must be used everywhere.

Reply:
Corrected the suggestion. Let me know if it works for you.

Reply:
updated